versioned_docs/version-8.7/self-managed/identity/configuration/identity-configuration-overview.md

```diff
@@ -0,0 +1,15 @@
+---
+id: identity-configuration-overview
+title: "Authentication, Keycloak and OIDC integration overview"
+sidebar_label: "Overview"
+description: "Learn more about how Identity plays together with Keycloak and other OIDC IdP applications to provide authentication services"
+---
+
+The default deployment of Camunda 8 self-managed uses Keycloak as IdP (Identity Provider) backend.
```

Comment:
We have "self-managed" three times on this page. Should be capitalized as "Self-Managed" as it's a proper name.

Reply:
Fixed by Mark